### ISSUE TITLE
repl::ReplSession::new function is private

### ISSUE THREAD

**Issue by yamaura** [zhiburt/expectrl#31]
Thanks for the wonderful library.
I want to create my custom ReplSession, but ReplSession::new function is private. So how can I create ReplSession from some sessions outside of this library?

I want to run program following:

```Rust
let session = expectrl::repl::ReplSession::new(
    expectrl::spawn("mycommand").unwrap(),
    "EXPECT_PROMPT".to_string(),
    None, false);
let recv = session.execute("send");
```

**Comment by yamaura** [comment]
Sorry for the lack of explanation.
Currently, I'm working on creating lib to log in automatically and then do one shot command.
I thought it would be easier if I could use ReplSession, which is already implemented since I would have to use some repl cli at the remote.

So really I want to like this:

```Rust
let session = expectrl::spawn("telnet remote").unwrap();
// session.expect("login:)....
// Do something log in procedure here
// It depends on remote machine, *nix, switches, special hardware and so on.
let session = expectrl::repl::ReplSession::new(session, "EXPECT_PROMPT".to_string(), None, false);
let recv = session.execute("send");
```

No problem, as I am in no hurry to publish it.
Thanks.

**Comment by zhiburt** [comment]
released in 0.4.0

please open an issue if there any more ones.

Thank you once again to addressing it.
